python/lsst/ts/linearstage/config_schema.py

```diff
@@ -12,80 +11,93 @@
 additionalProperties: false
 properties:
     stage_type:
-        description: Type of stage being controlled. Must be "Igus" or "Zaber"
+        description: Type of stage being controlled.
         type: string
-        default: 'Zaber'
+        enum: ["Zaber", "Igus"]
     target_position_minimum:
         description: >
           Minimum target position in mm
         type: number
-        default: 5
```

Comment:
If this is the most likely value please include it in the new description. Likewise for the other defaults you removed.

Also please use `description: >-` instead of `description: >` everywhere , because it gives better whitespace.